.yamato/scripts/update_from_upstream.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Prequisites:
+# This script assumes git and github CLR (gh) are installed.
+
+# Cause the script to fail if any individual command fails
+set -e
+
+# Fetch from upstream and rebase our local main on it
+git remote add upstream https://github.com/dotnet/runtime
+git fetch upstream
+git checkout --track origin/main
+git merge --ff-only upstream/main
+
+# Push our rebased local main to origin
+git push -f
```

Comment:
Should we have the `-f` here?  If we're `--ff-only` we shouldn't have to force push right?  And if `upstream/main` doesn't push cleanly we'd probably want the `push` to fail.

Reply:
Good point, we can remove `-f` here. I had that earlier due to the rebase. But we don't want that any more.

Reply:
Fixed now!